packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1231,7 +1232,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }, error);
 
         if (this.summaryManager !== undefined) {
-            this.summaryManager.off("summarizerWarning", this.raiseContainerWarning);
+            this.summaryManager.off("summarizerWarning", (warning: ContainerWarning) =>
```

Comment:
This doesn't work since it's creating a new anonymous function - you'll need to use the same function pointer in `.off()` as you used in `.on()` or it won't be able to find/remove the event listener.